mintpy/tsview.py

```diff
@@ -81,6 +84,21 @@ def create_parser():
     parser = arg_group.add_save_argument(parser)
     parser = arg_group.add_subset_argument(parser)
 
+    # temporal model fitting
+    model = parser.add_argument_group('deformation model', 'a suite of time functions')
+    model.add_argument('--model', dest='model_bool', type=bool, default=False,
+                      help='whether to view the ts2vel model fit (default: %(default)s).')
+    model.add_argument('--poly', '--polynomial', '--poly-order', dest='polynomial', type=int, default=1,
+                      help='a polynomial function with the input degree (default: %(default)s).')
+    model.add_argument('--periodic', '--period', '--peri', dest='periodic', type=float, nargs='+', default=[],
+                      help='periodic function(s) with period in decimal years (default: %(default)s).')
+    model.add_argument('--step', dest='step', type=str, nargs='+', default=[],
+                      help='step function(s) at YYYYMMDD (default: %(default)s).')
+    model.add_argument('--exp', '--exponential', dest='exp', type=str, nargs='+', action='append', default=[],
+                      help='exponential function(s) at YYYYMMDD with characteristic time(s) tau in decimal days (default: %(default)s).')
+    model.add_argument('--log', '--logarithmic', dest='log', type=str, nargs='+', action='append', default=[],
+                      help='logarithmic function(s) at YYYYMMDD with characteristic time(s) tau in decimal days (default: %(default)s).')
+
```

Comment:
With this usage, it is a good time to move them into a func in arg_group.py and re-used it in both ts2vel.py and tsview.py.

Reply:
Ok, I can do that.